geomstats/visualization.py

```diff
@@ -490,6 +494,186 @@ def draw(self, ax, **kwargs):
         ax.scatter(translation[:, 0], translation[:, 1], s=16, **kwargs)
 
 
+class KendallSphere:
+    """Class used to plot points in Kendall shape space of 2D triangles."""
+
+    def __init__(self, points=None, point_type='pre-shape'):
+        self.points = []
+        self.point_type = point_type
+        self.ax = None
+        self.elev, self.azim = None, None
+
+        # equilateral triangle (north pole)
+        self.can = gs.array([[1., 0.],
```

Comment:
What does the variable name `can` refer to? are C, A, N vertices? The name is not self-explanatory to me, maybe put something that is easier to understand?